README.md

```diff
@@ -18,6 +18,9 @@ https://docs.github.com/en/authentication/troubleshooting-ssh/error-permission-d
 
 *All combined notes, links and documents are listed below:
 
+ds-6-useful-links
+practicing 
```

Comment:
Lets remove the branch names and "test" or "practicing" messages within the documentation.